sci-libs/atlas/atlas-3.10.3.recipe

```diff
@@ -0,0 +1,145 @@
+SUMMARY="Automatically Tuned Linear Algebra Software"
+DESCRIPTION="A software library for linear algebra. It provides a mature open \
+source implementation of BLAS APIs for C and Fortran77."
+HOMEPAGE="http://math-atlas.sourceforge.net/"
+COPYRIGHT="2011-2016 The ATLAS Project"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/math-atlas/atlas$portVersion.tar.bz2"
+CHECKSUM_SHA256="2688eb733a6c5f78a18ef32144039adcd62fabce66f2eb51dd59dde806a6d2b7"
+SOURCE_DIR="ATLAS"
+PATCHES="atlas-$portVersion.patchset"
+ADDITIONAL_FILES="makefile.shared.mt
+	makefile.shared.st"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	atlas$secondaryArchSuffix = $portVersion
+	lib:libatlas$secondaryArchSuffix = $portVersion compat >=3
+	lib:libcblas$secondaryArchSuffix = $portVersion compat >=3
+	lib:libf77blas$secondaryArchSuffix = $portVersion compat >=3
+	lib:liblapack$secondaryArchSuffix = $portVersion compat >=3
+	lib:libptcblas$secondaryArchSuffix = $portVersion compat >=3
+	lib:libptf77blas$secondaryArchSuffix = $portVersion compat >=3
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgfortran$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	"
+CONFLICTS="
+	blas$secondaryArchSuffix
+	cblas$secondaryArchSuffix
+	lapack$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	atlas${secondaryArchSuffix}_devel = $portVersion compat >=3
+	devel:libatlas$secondaryArchSuffix = $portVersion compat >=3
+	devel:libcblas$secondaryArchSuffix = $portVersion compat >=3
+	devel:libf77blas$secondaryArchSuffix = $portVersion compat >=3
+	devel:liblapack$secondaryArchSuffix = $portVersion compat >=3
+	devel:libptcblas$secondaryArchSuffix = $portVersion compat >=3
+	devel:libptf77blas$secondaryArchSuffix = $portVersion compat >=3
+	"
+REQUIRES_devel="
+	atlas$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	binutils$secondaryArchSuffix
+	cmd:awk
+	cmd:bzip2
+	cmd:gfortran$secondaryArchSuffix
+	cmd:fgrep
```

Comment:
(This is a copy of one of the remarks I made on my previous review which is hiddent probably because the line to which it was attached was moved. It would not have happened if I had aborted my previous review (instead of just clicking "refresh") and rewritten it when I noticed there had been a new update to the PR. I'll be careful next time ;-)

You should probably use **`cmd:grep`** instead of **`cmd:fgrep`** because **fgrep is deprecated**, as we can see in this paragraph of the manual page:
```
       In addition, the variant programs egrep, fgrep and rgrep are
       the  same  as  grep -E,  grep -F, and grep -r, respectively.
       These variants are deprecated, but are provided for backward
       compatibility.
```
Of course, you should also replace every call to **`fgrep`** by **`grep -F`**.
BTW, this is the contents of **`/bin/fgrep`**:
```
#!/bin/sh
exec grep -F "$@"
```
Even if it's almost sure grep will continue to provide the fgrep variant, it should be avoided like any other deprecated stuff.